plugins/inputs/opcua/opcua_client.go

```diff
@@ -503,6 +503,7 @@ func disconnect(o *OpcUA) error {
 	case "opc.tcp":
 		o.state = Disconnected
 		o.client.Close()
+		o.client = nil
```

Comment:
Should we also be at least logging the return of o.client.Close()?

Reply:
Good question. I had a look at [the source code](https://github.com/gopcua/opcua/blob/baabb9b14fd272417c63c27cfce625a16c519518/client.go#L501) and it doesn't seem to have any code path that could actually produce errors.  Since `Close()` methods are usually called via `defer` and their return code ignored, I'd hazard a guess that this probably won't change anytime soon either.